Library/Formula/git-vendor.rb

```diff
@@ -0,0 +1,18 @@
+class GitVendor < Formula
+  desc "command for managing git vendored dependencies"
+  homepage "https://brettlangdon.github.io/git-vendor"
+  head "https://github.com/brettlangdon/git-vendor.git"
+
+  stable do
```

Comment:
Don't need a `stable do` block unless it requires something special/unique to stable, whether that be resources, patching, a dependency `head` doesn't, etc. If none of that is the case here, kill the block and wrap it into the initial block:

```ruby
desc ""
homepage ""
url ""
sha256 ""
head ""
```
etc.